### ISSUE TITLE
Unable to compile sumup SDK 2.4.1 with AndroidStudio 3.0.1

### ISSUE THREAD

**Issue by brinkkemper** [sumup/sumup-android-sdk#32]
Error:Execution failed for task ':abeldriver:javaPreCompileDebug'.
> Annotation processors must be explicitly declared now.  The following dependencies on the compile classpath are found to contain annotation processor.  Please add them to the annotationProcessor configuration.
    - argument-1.0.4.jar (com.neenbedankt.bundles:argument:1.0.4)
  Alternatively, set android.defaultConfig.javaCompileOptions.annotationProcessorOptions.includeCompileClasspath = true to continue with previous behavior.  Note that this option is deprecated and will be removed in the future.
  See https://developer.android.com/r/tools/annotation-processor-error-message.html for more details.

**Comment by vfoixcable** [comment]
Indeed, our SDK is not prepared for AS 3.0 yet. We already plan to work on fixing that but thanks for the head up anyway!
We will fix that with an upcoming version mid December

**Comment by brinkkemper** [comment]
ActionScript 3 support in December!? Awesome!
I'll start working on porting our ap to Flash :)

I do have to say thouh: AS3 came out in october. So fixing this in december is quite late.

**Comment by sullivan8** [comment]
Any news concerning AS3?

**Comment by vfoixcable** [comment]
Preparing the code base for AS3 was leading to breaking changes if a partner is not targeting a compileSDK and targetSDK >= than 23. 
Therefore, to deliver it properly, support for AS3 will come along SDK 3.0.0. We plan to release this SDK as part of Q1.

**Comment by vfoixcable** [comment]
as a quickfix, it seems that adding the following to the dependencies is fixing the issue:
```
annotationProcessor "com.neenbedankt.bundles:argument:1.0.4"
```

**Comment by epreviati** [comment]
Thanks @vfoixcable for the update.
I can confirm that currently your solution is temporary fixing compile errors on AndroidStudio 3.0.1 (with minSdk=21, targetSdk=27 and compileSdk = 27)

Thanks a lot!
